app/templates/admin.pug

```diff
@@ -48,6 +48,18 @@ block content
 
   if me.isOnlineTeacher() && !me.isAdmin()
     div.text-center Use the search box above to find and administer student accounts.
+  else if me.isParentAdmin() && !me.isAdmin()
+    p Set if admin is currently online / offline:
+    div.btn-group.btn-group-toggle#toggle-admin-availability(data-toggle="buttons")
+      label.btn.btn-secondary(data-value="on" class=`${parentAdminUpdateInProgress ? 'disabled' : ''} ${parentAdminAvailabilityStatus == 'on' ? 'active' : ''}`)
+        input(type="radio" name="options" autocomplete="off" :)
+        | Online
+      label.btn.btn-secondary(data-value="off" class=`${parentAdminUpdateInProgress ? 'disabled' : ''} ${parentAdminAvailabilityStatus == 'off' ? 'active' : ''}`)
+        input(type="radio" name="options" autocomplete="off")
+        | Offline
+      label.btn.btn-secondary(data-value="by-schedule" class=`${parentAdminUpdateInProgress ? 'disabled' : ''} ${parentAdminAvailabilityStatus == 'by-schedule' ? 'active' : ''}`)
+        input(type="radio" name="options" autocomplete="off")
+        | By Schedule
```

Comment:
It would be great to show `Saved` or `Success` on a change of schedule. I don't know if the action has happened on change right now.